ci/travis_jobs/docker_run_metplus.sh

```diff
@@ -22,6 +22,8 @@ echo "--Docker pull took $(($duration / 60)) minutes and $(($duration % 60)) sec
 
 echo CURRENT_BRANCH = ${CURRENT_BRANCH}
 
+docker pull ${DOCKERHUB_TAG}
```

Comment:
```suggestion
```

Why did you add another docker pull command here? The pull command is done before this line already.